src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -124,6 +125,14 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
 
             var antiforgeryDefault = true;
+            if (output.Attributes.ContainsName("method"))
```

Comment:
I take that back. Don't change the call to `GenerateForm()`. Instead unconditionally use `output.CopyHtmlAttribute()` for the (HTML) `method` attribute.